Assets/HoloToolkit/Input/Tests/Scripts/SphereKeywords.cs

```diff
@@ -1,26 +1,36 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
+using System;
 using UnityEngine;
 
 namespace HoloToolkit.Unity.InputModule.Tests
 {
-    public class SphereKeywords : MonoBehaviour
+    public class SphereKeywords : MonoBehaviour, ISpeechHandler
     {
         public void ChangeColor(string color)
         {
+            Material material = GetComponent<Renderer>().material;
```

Comment:
You should cache like this instead: ` renderer.sharedMaterial = _cachedMaterial;` and call it in Awake or Start